app/src/main/java/com/sourcepoint/test_project/MainActivity.java

```diff
@@ -1,14 +1,14 @@
 package com.sourcepoint.test_project;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.util.Log;
 import android.view.View;
 import android.view.ViewGroup;
 
+import androidx.appcompat.app.AppCompatActivity;
```

Comment:
why did we have to update the main activity with the appcompat? I'm not sure if this was necessary, but it is a nice addition indeed :)